sark-localbuild

```diff
@@ -189,6 +191,10 @@ if [ ${REPO_CLEAN_EXPIRE} -eq 1 ] ; then
   eit vacuum --quick --days ${REPO_CLEAN_EXPIRE_DAYS} ${REPOSITORY_NAME}
 fi
 
+if [[ ${REPO_BINHOST_CLEAN} -eq 1 ]] ; then
```

Comment:
-eq is for numeric values; use `=`

look:

```bash
$ [[ 0 -eq x ]] && echo "haha, yes!11!11"
haha, yes!11!11
```

Reply:
I'm confused. REPO_BINHOST_CLEAN variable is a number or at least could contains a number.
But also if it contains a wrong value or an empty value it works fine:

```bash
$# [[ "xxx" -eq 1 ]] && echo "Never printed"
$# [[ "0" -eq 1 ]] && echo "Never printed"
$# [[ "" -eq 1 ]] && echo "Never printed"
$# y="" ; [[ $y -eq 1 ]] && echo "Never printed"
$# y="1" ; [[ $y -eq 1 ]] && echo "Printed"
$# [[ "1" -eq 1 ]] && echo "Printed"
```

Why it's wrong use of *-eq* ? Is there some security reason ?

Reply:
Well actually, it's a matter of interpretation and your solution makes it simple: 1 enabled, something else (that doesn't become 1) disabled. I'm used to this: empty disabled, nonempty enabled, but this should be fine too.

Reply:
ok. np I can change it.
Personally, when there is an action that remove or clean files I prefer a conservative solution where I do that only if I have the right value. 
Thanks for clarification.

Reply:
And = will make it: only literal `1` would be matched, without doing any numeric conversion (-eq does).

(I did